Spinner/Spinner.js

```diff
@@ -159,14 +169,16 @@ const SpinnerBase = kind({
 	},
 
 	computed: {
-		className: ({children, color, size, transparent, styler}) => styler.append(
+		className: ({children, color, paused, size,  styler, transparent}) => styler.append(
```

Comment:
```suggestion
		className: ({children, color, paused, size,  styler, transparent}) => styler.append(
```
```suggestion
		className: ({children, color, paused, size, styler, transparent}) => styler.append(
```
two space between size and styler